lib/metrics.js

```diff
@@ -0,0 +1,46 @@
+const promClient = require('prom-client')
+const realtime = require('./realtime')
+
+exports.setupCustomPrometheusMetrics = function () {
+  const onlineNotes = new promClient.Gauge({
+    name: 'hedgedoc_online_notes',
+    help: 'Notes currently being edited'
+  })
+  const onlineSessions = new promClient.Gauge({
+    name: 'hedgedoc_online_sessions',
+    help: 'Sessions currently editing notes',
+    labelNames: ['type']
+  })
+  const onlineUsers = new promClient.Gauge({
+    name: 'hedgedoc_online_users',
+    help: 'Distinct users currently editing notes',
+    labelNames: ['type']
+  })
+  const notesCount = new promClient.Gauge({ name: 'hedgedoc_notes', help: 'Notes in the instance' })
```

Comment:
Any reason why this Gauge's config is inlined and not like the other ones split into multiple lines?

Reply:
I think the line is short enough to not make eslint unhappy